lib/matplotlib/axes/_axes.py

```diff
@@ -5173,8 +5182,8 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
         if im.get_clip_path() is None:
             # image does not already have clipping set, clip to axes patch
             im.set_clip_path(self.patch)
-        #if norm is None and shape is None:
-        #    im.set_clim(vmin, vmax)
+        # if norm is None and shape is None:
```

Comment:
You can just delete all the commented out code. It should not be there in the first place (but I am happy with the change).